packages/now-node/src/babel.ts

```diff
@@ -7,6 +7,7 @@ export function compile(
 ): { code: string; map: any } {
   return babel.transform(source, {
     filename,
+    configFile: false,
```

Comment:
Thanks for the PR! Can you add a test fixture to show that the failing use case is passing now?

Reply:
@styfle The test should fail since I removed `configFile: false` but it pass. Maybe the test system doesn't read babel config file?